### PR TITLE
Add recipe for asteroid-sounds

### PR DESCRIPTION
**Pull request by eLtMosen**
Sounds added to the src/ folder in asteroid-sounds repos shall end up in `/usr/share/sounds/`

### CODE REVIEW

File: recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
+HOMEPAGE = "https://github.com/AsteroidOS/asteroid-sounds"
+LICENSE = "CC0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
+
+SRC_URI = "git://github.com/AsteroidOS/asteroid-sounds;protocol=https;branch=master"
```
Comment (MagneFire) [inline recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb:6]: There's an error with this url.
The repo should be `git://github.com/AsteroidOS/asteroid-sounds.git`
And the `branch` should be `main` resulting in:
```
SRC_URI = "git://github.com/AsteroidOS/asteroid-sounds.git;protocol=https;branch=main"
```

File: recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
```
Comment (MagneFire) [inline recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb:1]: I don't think there's a need to mention in the summary under what license this is provided. Two lines below that's also listed :wink:

**Review by MagneFire — CHANGES_REQUESTED** [review]
PR looks good apart from the `SRC_URI` being incorrect and the minor comment about the license.

File: recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+HOMEPAGE = "https://github.com/AsteroidOS/asteroid-sounds"
+LICENSE = "CC0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
```
Comment (MagneFire) [inline recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb:4]: The license checksum isn't correct for what's provided in the repo. This should be `65d3616852dbf7b1a6d4b53b00626032`:
```
LIC_FILES_CHKSUM = "file://LICENSE;md5=65d3616852dbf7b1a6d4b53b00626032"
```

File: recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
+HOMEPAGE = "https://github.com/AsteroidOS/asteroid-sounds"
+LICENSE = "CC0"
```
Comment (MagneFire) [inline recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb:3]: `CC0` isn't a correct `SPDX` license (https://spdx.org/licenses/). Change it to `CC0-1.0`:
```
LICENSE = "CC0-1.0"
```

File: recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb (hunk 3 above)
Comment (eLtMosen) [reply recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb:4]: ugh, yeah, i did generate the md5sum for the meta-asteroid LICENSE files. Bravo to myself.

**Review by MagneFire — APPROVED** [review]
Perfect!

**Pull request merged**